app/scripts/services/events.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+angular.module('openshiftConsole').factory('EventsService', function() {
+
+  // TODO: decide if sessionStorage/localStorage should just be a service itself
+  var cacheKey = 'events-read';
```

Comment:
Let's prefix with `openshift/` per #1481. I realize it's session storage, but I think it's good to do anyway.